h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> What I don't like is talking about this current modification is if this was something unheard of,

I'm not fully aware of everything that the team has done for the past couple of years. At no point did I imply that this was (or wasn't) part of common practice. I just said I didn't like it. I'm reviewing the code I see, and expressing an opinion that there are better ways of doing what it does.

You can make up your own mind about the best solution here. I suspect that forking to our own repo is probably the cleanest option at this stage.